### PR TITLE
Completed es6 classes exercise. - Tiffany Brown

### PR DESCRIPTION
**Pull request by tiffanydbrown**
Screenshot 2023-11-21 at 3 16 20 PM
Screenshot 2023-11-21 at 3 15 59 PM
Screenshot 2023-11-21 at 3 08 03 PM

### CODE REVIEW

File: es6-classes/circle.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+class Circle extends Shape {
+  constructor(area, perimeter, radius) {
```
Comment (shawnkost) [inline es6-classes/circle.js:4]: This constructor should only be receiving 1 argument, for your `radius`.

File: es6-classes/circle.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    super(area, perimeter);
```
Comment (shawnkost) [inline es6-classes/circle.js:5]: Utilizing your `radius`, you need to calculate `area` & `perimeter`, and that's what you'll pass into `super`

File: es6-classes/square.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+class Square extends Shape {
+  constructor(area, perimeter, width) {
```
Comment (shawnkost) [inline es6-classes/square.js:4]: This constructor should only be receiving 1 argument, for your `radius`.

File: es6-classes/square.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    super(area, perimeter);
```
Comment (shawnkost) [inline es6-classes/square.js:5]: Utilizing your `width`, you need to calculate `area` & `perimeter`, and that's what you'll pass into `super`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Few changes here Tiffany!

**Comment by tiffanydbrown** [comment]
Screenshot 2023-11-21 at 3 46 55 PM
Screenshot 2023-11-21 at 3 46 35 PM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**